docs/gcp_appteam_scenario.md

```diff
@@ -0,0 +1,194 @@
+# Scenario: App team project creation
+
+The intent of this scenario is to show how an infrastructure team can use Compositions to create a new KCC namespace, and an associated GCP project for an application team. 
+
+## Facade
+
+The composition uses `AppTeam` CRD as its Facade (input schema). The facade CRD was created using [kubebuilder](https://book.kubebuilder.io/cronjob-tutorial/new-api). A developer using the `AppTeam`  CRD would not need to have any  knowledge of how the underlying KCC objects are defined. 
+
+```py
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  ...
+  name: appteams.idp.mycompany.com
+spec:
+  ...
+    kind: AppTeam
+  ...
+    schema:
+      openAPIV3Schema:
+  ...
+            properties:
+              project:  
+                type: string
+              adminUser:
+                type: string
+              billingUser:
+                type: string
+              folder:
+                type: string
+            required:
+            - project
+            - adminUser
+            - billingAccount
+            - folder
+ ...
+```
+
+## Composition definition
+
+The Composition and its corresponding Facade definition can be found [here](https://github.com/cloud-native-compositions/compositions/blob/main/samples/AppTeam/composition/appteam.yaml).   
+A preview of the `appteams` composition definition:
+
+```py
+apiVersion: composition.google.com/v1alpha1
+kind: Composition
+metadata:
+  name: appteams
+  namespace: default
+spec:
+  inputAPIGroup: appteams.idp.mycompany.com # inputAPIGroup specifies the CRD from which values are accepted as inputs into  this composition
```

Comment:
It is inputType and not group. A but we intended to fix but let is pass through !!

Reply:
For now it is an input resource. We will probably expand it to get the G K

Reply:
Lets open a bug and resolve the doc as well as the code+samples to be `inputResource` instead of `inputAPIGroup`. May be even split the input like this:

```
input:
   group: ...
   kind: ...
```